docs/api/symbol-package-publish-resource.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Push and Delete, NuGet API | Microsoft Docs
+author:
+- cristinamanu
+- kraigb
+ms.author:
+- cristinamanu
+- kraigb
+manager: skofman
+ms.date: 10/30/2018
+ms.topic: reference
+ms.prod: nuget
+ms.technology: null
+description: The publish service allows clients to publish new symbol packages.
+keywords: NuGet API push symbol package
+ms.reviewer:
+- karann
+- 
+---
+
+# Push Symbol Packages
+
+It is possible to push symbols packages ([snupkg](../create-packages/Symbol-Packages-V2.md)) using the NuGet V3 API.
+These operations are based off of the `PackagePublish` resource found in the [service index](service-index.md).
```

Comment:
The `docs/api/overview.md` file should be updated to mention this resource.

Reply:
Up[dated.